x-pack/plugins/uptime/public/components/queries/monitor_charts/monitor_charts_query.tsx

```diff
@@ -184,6 +184,7 @@ export class MonitorChartsQuery extends React.Component<Props, MonitorChartsStat
                           }
                         )}
                         data={downSeries}
+                        curve="curveBasis"
```

Comment:
Looks like this was just added for consistency?

Reply:
Yes, it's basically a tiny bug that I noticed while smoke testing my changes here. I can open a separate PR if you prefer.

Reply:
It's fine to leave it